ui/src/views/Settings.vue

```diff
@@ -19,15 +23,199 @@
+              <cv-dropdown-item  value="hr_HR">{{$t('settings.LOCALE_hr_HR')}}</cv-dropdown-item>
+              <cv-dropdown-item  value="hu_HU">{{$t('settings.LOCALE_hu_HU')}}</cv-dropdown-item>
+              <cv-dropdown-item  value="fr_FR">{{$t('settings.LOCALE_fr_FR')}}</cv-dropdown-item>
+            </cv-dropdown>
+            <NsComboBox
+              v-model.trim="timezone"
+              :autoFilter="true"
+              :autoHighlight="true"
+              :title="$t('settings.timezone')"
+              :label="$t('settings.choose_timezone')"
+              :options="accepted_timezone_list"
+              :userInputLabel="core.$t('settings.choose_timezone')"
+              :acceptUserInput="false"
+              :showItemType="true"
+              :invalid-message="$t(error.accepted_timezone_list)"
+              :disabled="loading.getConfiguration || loading.configureModule || loading.getDefaults"
+              tooltipAlignment="start"
+              tooltipDirection="top"
+              ref="accepted_timezone_list"
+            >
+              <template slot="tooltip">
+              {{
+                $t("settings.default_timezone_for_webtop_users")
+              }}
+              </template>
+            </NsComboBox>
+            <!-- advanced options -->
+            <cv-accordion ref="accordion" class="maxwidth mg-bottom">
+              <cv-accordion-item :open="toggleAccordion[0]">
+                <template slot="title">{{ $t("settings.advanced") }}</template>
+                <template slot="content">
+                  <cv-toggle
+                    value="webapp_debug"
+                    :label="$t('settings.webapp_debug')"
+                    v-model="webapp.debug"
+                    :disabled="loading.getConfiguration || loading.configureModule || loading.getDefaults"
+                    class="mg-bottom"
+                  >
+                    <template slot="text-left">{{
+                      $t("settings.disabled")
+                    }}</template>
+                    <template slot="text-right">{{
+                      $t("settings.enabled")
+                    }}</template>
+                  </cv-toggle>
+                  <NsByteSlider
+                    v-model="webapp.min_memory"
+                    :label="$t('settings.min_webapp_memory')"
+                    min="256"
+                    max="4096"
+                    step="1"
+                    stepMultiplier="1023"
+                    minLabel=""
+                    maxLabel=""
+                    :isUnlimited="false"
+                    :byteUnit="$t('settings.min_webapp_memory')"
+                    showHumanReadableLabel
+                    tagKind="high-contrast"
+                    :invalidMessage="$t(error.limit_min)"
+                    :disabled="loading.getConfiguration || loading.configureModule || loading.getDefaults"
+                    ref="min_memory"
+                  />
+                  <NsByteSlider
```

Comment:
As above: `showMibGibToggle` parameter is missing. I would set it to `true`

Reply:
I understand what you mean, my concern was that the unit of measure of the input is not very clear. I will enhance this on `NsByteSlider`: we'll show a label with *MiB* or *GiB* if `showMibGibToggle=false`. No changes required here